README.md

```diff
@@ -5,18 +5,43 @@
 The goal of this project is to automate preparation of downstream CI jobs
 for Red Hat OpenStack Platform (OSP) releases.
 
+The zuuler suite consists of the following tools:
```

Comment:
2nd point - will it be perhaps better to have zuuler <sub-command> format? this way I can run zuuler -h and see which tools are available for me, instead of reading the README.md or the code

Reply:
Hey, about the naming – let's discuss that on our team meeting. I'm not attached to the names, I just followed the original brainstormed draft, when we were not exactly sure how this will develop :-) I admit it became complex, but I think we should not exaggerate about forcing people to run `-h` – I bet the userbase will be a single digit number and the tool will end up as "set and forget", so one would have to dig anyway...
About 2nd point – by zuuler format do you mean to have a single CLI tool with subcommands for all the actions instead of separate commands? It makes sense to me, I add it to my TODO list.

Reply:
DONE